resources/js/Pages/Customer/Cart/Index.vue

```diff
@@ -0,0 +1,188 @@
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr v-for="(item, ind) in cart.items">
+                            <td scope="row">{{ ind+1 }}</td>
+                            <td><img src="/product-placeholder.png" alt="" width="150"></td>
+                            <td>{{ item.product.name }}</td>
+                            <td>{{ item.product.category.title }}</td>
+                            <td style="padding-right: .5rem; padding-left: .5rem"> <Quantity @change-val="changeTotal(ind)" v-model="item.quantity" /> </td>
+                            <!-- <td>{{ item.quantity }}</td> -->
+                            <td>{{ item.total.toFixed(2) }}</td>
+                            <td>
+                                <div class="d-flex" style="justify-content: center;">
+                                    <button @click="edit(item.id, ind)" class="edit-btn"><i class="fa-solid fa-edit"></i></button>
+                                    <button @click="destroy(item.id, ind)" class="delete-btn"><i class="fa-solid fa-xmark"></i></button>
+                                </div>
+                            </td>
+                        </tr>
+
+                        <p v-if="cart.items.length <= 0"> No item in the cart </p>
+                    </tbody>
+                </table>
+            </div>
+
+            <div class="receipt">
+                <p v-show="alert" style="font-weight: bold; color: green; margin-bottom: 1.75rem;"> Cart has been updated! </p>
+                <h3 style="margin-bottom: 60px;"> Receipt </h3>
+
+                <p> Total count of items: {{ quantity }} </p>
+                <p>Overall Total: {{ total }} </p>
+
+                <Link href="/cart/checkout"> 
+                    <ButtonIcon :disabled="cart.items.length < 1" color="green" style="padding-top: .25rem; padding-bottom: .25rem; float: right" label="Checkout" />
+                </Link>
+            </div>
+        </div>
+    </MainTemplate>
+</template>
+
+<script setup lang="ts">
+import { usePage, router, Link } from '@inertiajs/vue3';
+import MainTemplate from '@/MainTemplate.vue';
+import { Cart, CartItem } from '@/Utilities/Interfaces';
+import ButtonIcon from '@/Components/Button.vue'
+import Quantity from '@/Components/Quantity.vue'
+import { computed, reactive, ref } from 'vue';
+
+interface Item extends CartItem{
+    product: {
+        name: string,
+        price: number,
+        id: number
+        category: {
+            id: number,
+            title: string,
+        }
+    }
+}
+
+interface Current extends Cart{
+    items: Item[]
+}
+
+const alert = ref(false);
+
+const cart: Current = reactive(usePage().props.cart as Current);
+ 
+const total = computed (() => {
+    return cart.items.reduce( (sum, { total }) => (sum + total), 0).toFixed(2);
+})
+
+const quantity = computed (() => {
+    return cart.items.reduce( (sum, { quantity }) => (sum + quantity), 0)
+})
```

Comment:
Any data that needs to be computed before displaying is recommended to be done in the server-side. As a rule, it is recommended that all data sent to the frontend is already ready for displaying with little to no further manipulation needed.